src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -172,6 +169,11 @@
     <Compile Include="System\Transactions\DistributedTransactionPermission.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermissionAttribute.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard'">
+    <Compile Include="..\..\System.Runtime.Extensions\src\System\Security\IStackWalk.cs" />
```

Comment:
Should we instead move this files to common? That is the pattern we follow when shared across multiple projects.

Reply:
We have 250+ places where we do ` <Compile Include="..\..\`.

I think Common make sense for things that are really common within netcoreapp.

I do not think Common makes sense to share files for compat packages. These compat packages are transient. We are going to be building them for long - we dropped many of them already. Once we drop them, we would end up with stuff in Common that is not shared by anything.